jscomp/super_errors/super_typetexp.ml

```diff
@@ -109,8 +109,8 @@ let report_error env ppf = function
       fprintf ppf "@[<v>\
       @{<info>The variant constructor %a can't be found.@}@,@,\
       @[<v 2>- If it's defined in another module or file, bring it into scope by:@,\
-        @[- Annotating it with said module name:@ @{<info>let food = MyModule.Apple@}@]@,\
-        @[- Or specifying its type:@ @{<info>let food: MyModule.fruit = Apple@}@]\
+        @[- Prefixing it with said module name:@ @{<info>TheModule.%a@}@]@,\
```

Comment:
how does this work if you don't add what you wanna print as additional arguments to `fprintf`?

Reply:
See my edit ;-)